conform-pr/main.go

```diff
@@ -128,14 +145,13 @@ type pullRequest struct {
 func (pr *pullRequest) checkTitle() error {
 	match, err := regexp.MatchString("[a-zA-Z0-9`'\"]$", pr.title)
```

Comment:
If we use `MustCompile` there, this method could only return an error if title is invalid

Reply:
Yes, sounds good!

Reply:
applied to both.